yktracec/src/jitmodbuilder.cc

```diff
@@ -1732,6 +1736,10 @@ class JITModBuilder {
             }
             CallStack.pushFrame(StackFrame::CreateForeignFrame());
             break;
+          } else if (CF->getName() == PromoteRecFnName) {
+            // A variable is being promoted to a constant.
```

Comment:
s/variable/value/

Reply:
This one is actually fine since we are working at the LLVM level.

In LLVM prose `Value` is the type of an "SSA variable". I'll update it anyway.